smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/SmokeTestGenerator.kt

```diff
@@ -53,10 +56,10 @@ open class SmokeTestGenerator(
      */
     private fun getOperationShapeIdToTestCasesMapping(serviceName: String): Map<ShapeId, List<SmokeTestCase>> {
         val operationShapeIdToTestCases = mutableMapOf<ShapeId, List<SmokeTestCase>>()
-        ctx.service.allOperations.forEach { op ->
-            if (ctx.model.expectShape(op).hasTrait<SmokeTestsTrait>()) {
+        ctx.model.operationShapes.forEach { op ->
```

Comment:
Smoke test codegen was happening only for operations directly bound to service. The change here fixes the bug to codegen smoke tests even for operations bound to nested resource shapes of a service.